cli/tools/repl/mod.rs

```diff
@@ -123,10 +163,47 @@ pub async fn run(
     }
   }
 
+  Ok(repl_session)
+}
+
+fn save_session_to_file(
+  session_history: &[String],
+  maybe_filename: Option<String>,
+) -> Result<(), AnyError> {
+  // TODO(bartlomieju): make date shorter
+  let filename = maybe_filename.unwrap_or_else(|| {
+    format!("./repl-{}.ts", chrono::Local::now().to_rfc3339())
+  });
+  std::fs::write(&filename, session_history.join("\n"))
+    .context("Unable to save session file")?;
+  println!("Saved session to {}", filename);
```

Comment:
Is the filename here clickable? Maybe we should show the full filepath?

Reply:
Good point, I'll check how to do it

Reply:
Yeah, it's clickable